Minesweeper.py

```diff
@@ -43,68 +28,86 @@
                 print(board[y][x], end=" ")
         print()
 
-    # Print the game stats
+def print_stats():
     print("Moves:", moves)
     print("Flags:", flags)
     print("Mines found:", mines_found)
 
-    # Get the player's input
-    action = input("Enter your action (move, flag, or unflag): ")
-    direction = input("Enter the direction (up, down, left, or right): ")
-
-    # Update the player's position
+def update_player_pos(direction):
+    x, y = player_pos
     if direction == "up":
-        player_y -= 1
+        y -= 1
     elif direction == "down":
-        player_y += 1
+        y += 1
     elif direction == "left":
```

Comment:
There's an indentation issue in the `update_player_pos` function that makes the code logic unclear. However, it seems like the function intends to branch based on the direction input.
```diff
-     if direction == "up":
+    if direction == "up":
-      elif direction == "down":
+    elif direction == "down":
-      elif direction == "left":
+    elif direction == "left":
-      elif direction == "right":
+    elif direction == "right":
```
---